packages/uniforms-bootstrap3/src/ListAddField.tsx

```diff
@@ -36,6 +36,8 @@ function ListAdd({
     !disabled && !(parent.maxCount! <= parent.value!.length);
 
   return (
+    // TODO[jsx-a11y]
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
```

Comment:
I think we could disable them globally in `.eslintrc` for now - this makes too much noise.